### DISCUSSION TITLE
How do you use a new Embedding

### DISCUSSION THREAD

**Discussion by tonym1995** [betalgo/openai#159] [Q&A]
Hi All,

I've successfully created an embedding using the code example. Although I loaded the data from a text file and got 20000+ vectors in the data.

So, how do I use that embedding as a model for Q & A. I've successfully used the same data in python to create a custom model so now I would like to do the same thing in C#. It will be used as a desktop application specific chat assistant.

Any help would be greatly appreciated.

Thanks

**Comment by kayhantolga** [comment]
I was attempting to create a sample, but it has not finished yet :/
